package.json

```diff
@@ -25,10 +25,10 @@
  "scripts": {
   "start": "electron .",
   "postinstall": "electron-builder install-app-deps",
-  "build:windows": "electron-builder --windows",
-  "build:mac": "electron-builder --mac",
-  "build:linux": "electron-builder --linux",
-  "release": "electron-builder --mac --windows --linux --publish always"
+  "build:windows": "electron-builder --windows --x64 --arm64",
```

Comment:
I would prefer that you remove all of the --x64 and --arm64 from the windows, mac and linux commands. Then add two new scripts for build:x64 and build:arm64 respectively. the release script changes can stay.

Reply:
Will do!